packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -0,0 +1,63 @@
+import { IResource, Resource } from 'aws-cdk-lib';
+import { CfnSchedule } from 'aws-cdk-lib/aws-scheduler';
+import { Construct } from 'constructs';
+import { ScheduleExpression } from './schedule-expression';
+
+/**
+ * DISCLAIMER: WORK IN PROGRESS, INTERFACE MIGHT CHANGE
+ * 
+ * This unit is not yet finished. Only rudimentary Schedule is implemented in order
+ * to be able to create some sensible unit tests
+ */
```

Comment:
Agree that this is possible. I moved the other stuff except for the interface ISchedule to the private directory. I do not agree though that this was the best way forward since this all needs to be undone in the coming PRs. It seems an unnecessary hassle for an experimental module.

I wanted to introduce these incomplete-but-working classes (Schedule and Target) to also be able to add a working integration test in this PR with the implemented features. For now we did not test that the currently implemented features Expression and Input actually work through deployment with the generated CloudFormation.

Also the rosetta tests currently break because the examples reference Schedule.